en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/event-hub.md

```diff
@@ -1,10 +1,13 @@
 # Event Hub
 
-Choreo Connect uses the below methods to fetch data from API Manager.
+The Event Hub acts as a mediator between the adapter and the control plane. Choreo Connect uses the below methods to fetch data from API Manager.
 
 - Internal Data REST API
 - JMS Topic
 
+!!! note
+    The event hub is included in the traffic manager node of APIM. Therefore, if you use a distributed setup use the ip address of the traffic manager node for the configurations.
```

Comment:
link the doc here as well https://apim.docs.wso2.com/en/latest/install-and-setup/setup/distributed-deployment/product-profiles/

Reply:
Added with https://github.com/wso2/docs-apim/pull/3839/commits/618a329d1dd1ef57172162610208c09d8c6d4c99